content/logs/aws.md

```diff
@@ -28,30 +27,30 @@ Using a Lambda function, called whenever a new S3 log file is made available, av
 
 #### Provide the code and configure the lambda
 
-6. Copy and paste the code from [This repo](https://github.com/DataDog/dd-aws-lambda-functions/blob/master/Log/lambda_function.py) into the function code area.
-7. Change the Runtime to **Python 2.7**
+1. Copy and paste the code from [this repo](https://github.com/DataDog/dd-aws-lambda-functions/blob/master/Log/lambda_function.py) into the function code area.
+2. Change the Runtime to **Python 2.7**
```

Comment:
Let's add a section for the handler (screenshot provided to @l0k0ms):

And set the handler to `lambda_function.lambda_handler`